### PR TITLE
Feature/imu

### PR DESCRIPTION
**Pull request by masonturner**

### CODE REVIEW

File: embedded/gpio/gpio_test.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// Build with gcc -o gpio_test gpio_test.c -lwiringPi -lwiringPiDev -lpthread -lcrypt -lm -lrt
+#include <wiringPi.h>
```
Comment (jpanikulam) [inline embedded/gpio/gpio_test.c]: Delete this?

File: third_party/bno055/Example_Data_Raw.cpp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+	    /* Display the floating point data */
+	    std::cout << "X: " << euler.x() <<    " Y: " << euler.y() << " Z: "
+		<< euler.z() << "\t\t\n\r";
+
+	    /*
+	    // Quaternion data
+	    imu::Quaternion quat = bno.getQuat();
+	    Serial.print("qW: ");
+	    Serial.print(quat.w(), 4);
```
Comment (jpanikulam) [inline third_party/bno055/Example_Data_Raw.cpp]: Delete commented code

File: embedded/gpio/gpio_test.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// Build with gcc -o gpio_test gpio_test.c -lwiringPi -lwiringPiDev -lpthread -lcrypt -lm -lrt
+#include "third_party/wiringpi/wiringPi.h"
+ 
+int main(void)
+{
+    wiringPiSetup();
+    pinMode(0, OUTPUT);
+ 
+    for (;;)
```
Comment (jpanikulam) [inline embedded/gpio/gpio_test.c]: Do while

Comment (jpanikulam) [reply embedded/gpio/gpio_test.c]: Use `while` instead of this weird thing

Comment (masonturner) [reply embedded/gpio/gpio_test.c]: Done.

**Pull request merged**